apps/trustlab/src/components/DonorOverview/DonorOverview.js

```diff
@@ -0,0 +1,99 @@
+import { Section } from "@commons-ui/core";
+import { Figure, Link, RichTypography } from "@commons-ui/next";
+import { Button, Grid } from "@mui/material";
+import React from "react";
+
+const DonorOverview = React.forwardRef(function DonorOverview(props, ref) {
+  const { sx, partners, title, action } = props;
```

Comment:
```suggestion
const DonorOverviewList = React.forwardRef(function DonorOverviewList(props, ref) {
  const { sx, donors, title, action } = props;
```
1. Since it's multiple, it should have `List` in name (or `Grid`, etc.)
2. Donor component can not receive `partners`. It should be `donors` or `items` if we want generic prop.